README.md

```diff
@@ -7,8 +7,16 @@
 [![Code Style: Blue](https://img.shields.io/badge/code%20style-blue-4495d1.svg)](https://github.com/invenia/BlueStyle)
 [![ColPrac: Contributor's Guide on Collaborative Practices for Community Packages](https://img.shields.io/badge/ColPrac-Contributor's%20Guide-blueviolet)](https://github.com/SciML/ColPrac)
 
-FeatureTransforms.jl provides utilities for performing feature engineering in machine learning pipelines.
-FeatureTransforms supports operations on `AbstractArrays` and [Tables](https://github.com/JuliaData/Tables.jl).
+FeatureTransforms.jl provides utilities for performing feature engineering in machine learning pipelines with support for `AbstractArray`s and [`Table`](https://github.com/JuliaData/Tables.jl)s.
+
+## Getting Started
+
+There are a few key parts to the Transforms.jl API, refer to the documentation for each to learn more.
+
+1. [`Transform`](@ref about-transforms)s are callable types that define a certain operation to be performed on data. For example, normalizating or a computing a linear combination. Refer to the [page about Transforms](@ref about-transforms) to learn how they are defined and used on various types of input.
```

Comment:
Broken links https://github.com/invenia/FeatureTransforms.jl/tree/gm/docs

Reply:
I guess they don't render in the README then

Reply:
Yeah, we could link directly with URL?